README.md

```diff
@@ -4,133 +4,64 @@
 *WARNING*: The [version v9.x](https://github.com/topfreegames/maestro/tree/v9) of Maestro is under deprecation, complete guide of the new version v10.x can be found [here](https://github.com/topfreegames/maestro/issues/283).
 
 ---
-Maestro: Kubernetes Game Room Scheduler 
-=======================================
+# Maestro 
 
 [![Build Status](https://github.com/topfreegames/maestro/actions/workflows/test.yaml/badge.svg?branch=next)](https://github.com/topfreegames/maestro/actions/workflows/test.yaml)
 [![Codecov Status](https://codecov.io/gh/topfreegames/maestro/branch/next/graph/badge.svg?token=KCN2SZDRJF)](https://codecov.io/gh/topfreegames/maestro)
 
-## Docs
-All documentation regarding this version (v10.x, AKA NEXT) can be accessed at https://topfreegames.github.io/maestro/.
+[//]: # (One or two sentences description of the module/service in non technical terms and information about the owner. Text in quotes are examples.)
+
+Game room management service, designed to manage multiple game room fleets in Kubernetes clusters.
```

Comment:
When I came to the maestro team, we had a discussion about Kubernetes being connected to the maestro. AFAIK, the Kubernetes is currently only an implementation of the runtime, which could be changed to another in the future. What do you think about removing the last part?